CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# How to Contribute
+
+Information on contributing to this repo is in the [Contributing Guide](https://github.com/AndcultureCode/AndcultureCode/blob/master/CONTRIBUTING.md) in the Home repo.

```

Comment:
I think this will automatically redirect to the default branch if the specified branch cannot be found, so it'll still work - but we should probably update this to point to `main` regardless.